docs/src/AlgebraicGeometry/Schemes/MorphismsOfProjectiveSchemes.md

```diff
@@ -22,9 +22,9 @@ out.
 
 ## Constructors
 ```@docs
-    morphism_of_projective_schemes(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map; check::Bool=true )
-    morphism_of_projective_schemes(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map, h::SchemeMor; check::Bool=true )
-    morphism_of_projective_schemes(X::AbsProjectiveScheme, Y::AbsProjectiveScheme, a::Vector{<:RingElem})
+    morphism(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map; check::Bool=true )
+    morphism(P::AbsProjectiveScheme, Q::AbsProjectiveScheme, f::Map, h::SchemeMor; check::Bool=true )
+    morphism(X::AbsProjectiveScheme, Y::AbsProjectiveScheme, a::Vector{<:RingElem})
```

Comment:
@simonbrandhorst : Do you know when this happened? What is the anticipated format? Didn't we decide for plain `morphism` at some point?

Reply:
no clue, but we did decide for morphism

Reply:
Needs a deprecation though. The function is exported and in src.